### DISCUSSION TITLE
Migrating to freya with java 8 and primefaces 10

### DISCUSSION THREAD

**Discussion by maikefb** [primefaces/community#1499] [PrimeFaces Templates] [answered]
Hello.
I'm working on a legacy system using Java 8 and Primefaces 8 with the Modena theme
I started upgrading Primefaces to 10. Medena does not accept this version of Primefaces.
So I intend to buy freya to continue the migration, but it shows primefaces 13 as a requirement
Is there an older version of Freya that accepts Primefaces 10?

Because it's a very legacy system, I'm making the transition in parts and I wouldn't like to jump from Primefaces 8 to 13.

Thank you for your attention

**Comment by melloware** [comment, marked as answer]
@maikefb if you DM me on the Discord channel i have a version of Modena i upgraded to PF13/14 for another client.

**Comment by tandraschko** [comment]
Plese see: https://x.com/primefaces/status/1970401400568918449
